README.md

```diff
@@ -46,10 +46,27 @@ unleash.on('synchronized', () => {
   const variant = unleash.getVariant('app.ToggleY');
 });
 ```
+Be aware that the `initialize` function will configure a _global_ Unleash instance. If you call this method multiple times the global instance will be changed. 
 
-Be aware that the `initialize` function will configure a _global_ Unleash instance. If you call this
-method multiple times the global instance will be changed. If you prefer to handle the instance
-yourself you should [construct your own Unleash instance](#alternative-usage).
+#### <u>Constructing the Unleash client directly</u>
+If you prefer to construct the Unleash instance yourself rather than using the global `initialize` method you can do so e.g.:
```

Comment:
Nice! When would you want to do this / what problems does it solve? I think giving the users an example of when they'd want to do this would be useful 🤓

Also, a slight rewording suggestion:
```suggestion
You can also construct the Unleash instance yourself instead of via the `initialize` method. 
This lets you [...] // <- why is this useful?
```

Reply:
Yeah the idea was to stylize it so it looked like a proper subsection of section `2.1`, but not that important anyways 👍🏽 
My markdown renderer locally did render the underline properly so  i thought it would work on Github too 🤷🏽